storybookjs/storybook
# CLI: Run `storybook dev` as part of `storybook init`

Closes #22871



## What I did

- add --initial-path option to the CLI
- run storybook dev as part of storybook init

## How to test

1. Build all packages
2. Create a new project with the CLI e.g. `./code/lib/cli/bin/index.js init --initial-path=/settings/shortcuts`
3. Expect Storybook to open in that route

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
